core/src/main/java/org/frankframework/json/JsonUtil.java

```diff
@@ -57,8 +57,6 @@ private JsonUtil() {
 	public static JsonMapper buildJsonMapper(IScopeProvider scopeProvider, String stylesheetName, DataSonnetOutputType outputType, boolean computeMimeType, ParameterList parameters) throws ConfigurationException {
 		try {
 			return new JsonMapper(getStyleSheet(scopeProvider, stylesheetName), outputType, computeMimeType, parameters.getParameterNames());
-		} catch (ConfigurationException e) {
-			throw e;
```

Comment:
We hoeven deze exception niet af te vangen alleen maar om hem te re-throwen, want er is geen andere catch die een parent-class exception afvangt.